Makefile test target fails

# Issue
`make test` does not run the test suite.

## Command run
```
$ make test
```

## Output
```
$ make test
core/bindata.go:1:1: package comment should be of the form "Package core ..."
core/bindata.go:203:5: don't use underscores in Go names; var _staticImagesGopher_fullPng should be _staticImagesGopherFullPng
core/bindata.go:205:6: don't use underscores in Go names; func staticImagesGopher_fullPngBytes should be staticImagesGopherFullPngBytes
core/bindata.go:212:6: don't use underscores in Go names; func staticImagesGopher_fullPng should be staticImagesGopherFullPng
core/bindata.go:223:5: don't use underscores in Go names; var _staticImagesGopher_headPng should be _staticImagesGopherHeadPng
core/bindata.go:225:6: don't use underscores in Go names; func staticImagesGopher_headPngBytes should be staticImagesGopherHeadPngBytes
core/bindata.go:232:6: don't use underscores in Go names; func staticImagesGopher_headPng should be staticImagesGopherHeadPng
core/bindata.go:263:5: var _staticIndexHtml should be _staticIndexHTML
core/bindata.go:265:6: func staticIndexHtmlBytes should be staticIndexHTMLBytes
core/bindata.go:272:6: func staticIndexHtml should be staticIndexHTML
core/bindata.go:403:5: don't use underscores in Go names; var _staticJavascriptsHighlight_workerJs should be _staticJavascriptsHighlightWorkerJs
core/bindata.go:405:6: don't use underscores in Go names; func staticJavascriptsHighlight_workerJsBytes should be staticJavascriptsHighlightWorkerJsBytes
core/bindata.go:412:6: don't use underscores in Go names; func staticJavascriptsHighlight_workerJs should be staticJavascriptsHighlightWorkerJs
core/bindata.go:483:5: var _staticStylesheetsApplicationCss should be _staticStylesheetsApplicationCSS
core/bindata.go:485:6: func staticStylesheetsApplicationCssBytes should be staticStylesheetsApplicationCSSBytes
core/bindata.go:492:6: func staticStylesheetsApplicationCss should be staticStylesheetsApplicationCSS
core/bindata.go:503:5: var _staticStylesheetsBootstrapCss should be _staticStylesheetsBootstrapCSS
core/bindata.go:505:6: func staticStylesheetsBootstrapCssBytes should be staticStylesheetsBootstrapCSSBytes
core/bindata.go:512:6: func staticStylesheetsBootstrapCss should be staticStylesheetsBootstrapCSS
core/bindata.go:523:5: var _staticStylesheetsHighlightCss should be _staticStylesheetsHighlightCSS
core/bindata.go:525:6: func staticStylesheetsHighlightCssBytes should be staticStylesheetsHighlightCSSBytes
core/bindata.go:532:6: func staticStylesheetsHighlightCss should be staticStylesheetsHighlightCSS
core/bindata.go:543:5: var _staticStylesheetsOpeniconicCss should be _staticStylesheetsOpeniconicCSS
core/bindata.go:545:6: func staticStylesheetsOpeniconicCssBytes should be staticStylesheetsOpeniconicCSSBytes
core/bindata.go:552:6: func staticStylesheetsOpeniconicCss should be staticStylesheetsOpeniconicCSS
core/git.go:268:3: var baseUrl should be baseURL
core/git.go:269:3: var uploadUrl should be uploadURL
/bin/bash: line 1: cd: ./...wraith: No such file or directory
make: *** [Makefile:88: test] Error 1
$ pwd
.../go/src/github.com/shreyas-sriram/wraith
```

# Possible Fix
Is it required to run `@cd ./...$(pkg)`? Can we do `go test -cover ./...` directly instead?

## Output
```
$ go test -cover ./...
?   	github.com/N0MoreSecr3ts/wraith	[no test files]
?   	github.com/N0MoreSecr3ts/wraith/cmd	[no test files]
ok  	github.com/N0MoreSecr3ts/wraith/core	0.018s	coverage: 1.0% of statements
?   	github.com/N0MoreSecr3ts/wraith/version	[no test files]
```

--- a/Makefile
+++ b/Makefile
@@ -85,5 +85,5 @@ pretty:
 
 ## test		Run tests with coverage
 test: pretty
-	@cd ./...$(pkg) && go test -cover
+	go test ./... -cover
 

